org.mozilla.Firefox.appdata.xml

```diff
@@ -1,13 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
-  <id>org.mozilla.Firefox.desktop</id>
+  <id>org.mozilla.Firefox</id>
+  <launchable type="desktop-id">org.mozilla.Firefox.desktop</launchable>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>MPL-2.0</project_license>
 
-  <categories>
-    <category>Utility</category>
-  </categories>
-
```

Comment:
Why delete this?

Reply:
Apparently now the preferred way is to leave categories inside the .desktop file, and they'll automatically be pulled from there instead of duplicating them here too.

Reply:
Yeah, appstream-compose handles this.

Reply:
Ack.